### ISSUE TITLE
Cross icon

### ISSUE THREAD

**Issue by pauloneillft** [Financial-Times/o-overlay#32]
Currently is an image. Agreed today to change to an font character icon.
@AlbertoElias

**Comment by kaelig** [comment]
Thanks for letting me know. Giving it a shot and will keep you up to date.

**Comment by kaelig** [comment]
The only way I can see to have an entirely coherent rendering across platforms in a retro-compatible way (without pushing a major version of this module) is to use pseudo elements to build a `+` sign, and then orienting the object 45 deg with CSS transforms to make it look like a `×`. 


Since IE < 9 doesn't support CSS transforms, we need to use a filter.

Using this [matrix calculator](http://www.boogdesign.com/examples/transforms/matrix-calculator.html), we get:
```css
-ms-filter: "progid:DXImageTransform.Microsoft.Matrix(M11=0.70710678, M12=-0.70710678, M21=0.70710678, M22=0.70710678,sizingMethod='auto expand')";
filter: progid:DXImageTransform.Microsoft.Matrix(M11=0.70710678, M12=-0.70710678, M21=0.70710678, M22=0.70710678,sizingMethod='auto expand');
```

**Comment by triblondon** [comment]
Why don't we combine the X with the circle in a single icon?  The image service filter function (once fixed to work with icons) would then give you the ability to generate a sprite for legacy support.

**Comment by AlbertoElias** [comment]
We do have three different colours there, the border, circle background and
the x. And they have different states, and also shaded/non-shaded. If we
had a single icon, do you mean an svg where we would colour the different
paths via css?

-- 

------------------------------
This email was sent by a company owned by Pearson plc, registered office at 
80 Strand, London WC2R 0RL.  Registered in England and Wales with company 
number 53723.

**Comment by kaelig** [comment]
An other solution would be to force the design to only have one icon, no hover state. Then it would be easy to provide a SVG icon with a PNG fallback.

**Comment by kaelig** [comment]
Just had some fun… and made the cross in CSS.

http://codepen.io/kaelig/pen/yyMEEE

### Rendering in IE 8:

![screenshot 2015-01-15 18 45 40](https://cloud.githubusercontent.com/assets/85783/5764050/004a035c-9ce7-11e4-9d1d-6c8ddc815543.png)


### Rendering in IE 9

![screenshot 2015-01-15 18 48 17](https://cloud.githubusercontent.com/assets/85783/5764067/216bb24c-9ce7-11e4-8a8e-5ed8039a9f97.png)


@AlbertoElias do you want to have a go at putting that cross in the module?

**Comment by kaelig** [comment]
Fixed in #47 & release 1.3.0
